sale_payment_term_interest/tests/test_payment_term.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+#
+#
+#    Authors: Guewen Baconnier
+#    Copyright 2015 Camptocamp SA
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+#
+
+from operator import itemgetter
+from openerp.tests import common
+
+
+class TestPaymentTerm(common.TransactionCase):
+
+    def test_interest(self):
+        term = self.env['account.payment.term'].create({'name': 'test'})
+        self.env['account.payment.term.line'].create({
+            'payment_id': term.id,
+            'value': 'procent',
+            'value_amount': 0.3333,
+            'days': 10,
+            'interest_rate': 0.0,
+        })
+        self.env['account.payment.term.line'].create({
+            'payment_id': term.id,
+            'value': 'procent',
+            'value_amount': 0.3333,
+            'days': 40,
+            'interest_rate': 0.041666,
```

Comment:
for readability purpose, it would be nicer in my opinion to have this written as 15/(12*30) (and to add `from __future__ import division`) at the beginning of the file (and I still feel it would be more user friendly to have 15 for the value here).